feconf.py

```diff
@@ -1295,3 +1295,7 @@ def get_empty_ratings():
     SUGGESTION_TYPE_TRANSLATE_CONTENT,
     SUGGESTION_TYPE_ADD_QUESTION
 ]
+
+# Minimum timestamp difference required between UserSettingsModel's created_on
+# and some other attribute to use it for updating the created_on attribute.
+CREATED_ON_TIME_DELTA_MINUTES = 5
```

Comment:
Why have it here, move it to _user_jobs_one_off.py_

Reply:
Done now. So basically only the constants which could be used at "many" places in the backend should be kept here right, and not all the constants?